_posts/2023-10-08-amazon_bedrock_part2.markdown

```diff
@@ -0,0 +1,130 @@
+
+All these amazing features and optimizations are like extra layers of icing on the cake, but for our project, we're keeping things focused. So, while they're fascinating possibilities, we'll save them for another time!
+
+## Build an Storytelling app with me 
+
+Take a look at the cool diagram below! It shows you exactly how our app works at every step:
+
+- The user, starts by giving us an idea for a story or what the story should be about.
+
+- When user clicks on "Generate Story," the web app send a request to foundation model to create the story, and then it returns the generated story.
+
+- Now, here's where it gets interesting. user can either generate another story with the same topic or change the topic and get a new one. Plus, they can even add illustrations to the story! In this app, I've configured the FM model to generate an image for each paragraph. These images are stored in an S3 bucket, and the UI shows them to user once it gets back the S3 presigned URLs.
+
+For all the nitty-gritty details, just check out the solution architecture diagram. It's like a map that guides you through the app's awesomeness.
+
+
+### Architecture
+
+![Solution Diagram](/img/bedrock/solution_diagram.png)
+
+### Web application
+
+We won't be diving into the fancy frontend design here. I've laid out the basic structure of the application. You can grab the source code from our [GitHub repository](#) and give it your own unique style or add more features if you'd like. Once you've got the code, just follow the simple steps in the ReadMe file to get your app running on your computer.
+
+And if you're feeling adventurous and want to share your app with the world, you can even host it on your AWS account. I won't get into the nitty-gritty details in this blog post, but all you really need is an Amazon S3 bucket to store your web app's resources. Then, set up Amazon CloudFront and use Route 53 to manage your domain's traffic and routing. It's not as complicated as it might sound, and it's a great way to take your project to the next level!
+
+
+### Amazon Bedrock Magician
+
+To set up the necessary APIs for our app to function, we'll be creating a serverless stack. You can access the complete source code in the [GitHub repository]() within the "Backend" folder. In this folder, you'll find all the required Functions, IAM Roles, and Managed Policies listed in the "serverless.yml" file. To deploy the API, all you have to do is run the command specified in the "ReadMe.md" file.
```

Comment:
hmm missing the repository url in `[GitHub repository]()`?

Reply:
Nah I will provide it shortly

Reply:
Crazy